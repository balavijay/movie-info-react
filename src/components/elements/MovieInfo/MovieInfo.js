import React from 'react';
import PropTypes from 'prop-types';

import NoImage from '../../images/no_image.jpg';
import { IMAGE_BASE_URL, POSTER_SIZE } from '../../../config';

import MovieThumb from '../MovieThumb/MovieThumb';

import { StyledMovieInfo } from '../../styles/StyledMovieInfo';

const MovieInfo = ({ movie }) => (
  <StyledMovieInfo >
    <div className="movieinfo-content">
      <div className="movieinfo-thumb">
        <MovieThumb
          image={
            movie.poster_path
              ? `${IMAGE_BASE_URL}${POSTER_SIZE}${movie.poster_path}`
              : NoImage
          }
          clickable={false}
          alt="moviethumb"
        />
      </div>
      <div className="movieinfo-text">
        <h1>{movie.title}</h1>

     

      </div>
    </div>
  </StyledMovieInfo>
);

MovieInfo.propTypes = {
  movie: PropTypes.object,
  directors: PropTypes.array,
}

export default MovieInfo;
