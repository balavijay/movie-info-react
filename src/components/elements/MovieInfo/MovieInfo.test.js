import React from 'react';
import { shallow } from 'enzyme';


import MovieInfo from './MovieInfo';

describe ("MovieInfo Component " , () => {
    let movie = {
        "poster_path" : ""
    }
   const  app = shallow(<MovieInfo movie={movie} />);
    test("MovieInfo render" , () => {
        expect(app).toMatchSnapshot();
    })

 
});