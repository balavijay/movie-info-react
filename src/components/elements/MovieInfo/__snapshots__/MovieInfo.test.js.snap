// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieInfo Component  MovieInfo render 1`] = `ShallowWrapper {}`;
